hasty/image.py

```diff
@@ -171,7 +171,7 @@ def get_labels(self):
                              obj_params={"project_id": self.project_id})
 
     def create_label(self, label_class: Union[LabelClass, str], bbox: List[int] = None, polygon: List[List[int]] = None,
-                     mask: List[int] = None, z_index: int = None, external_id: Optional[str] = None):
```

Comment:
@kostya-cf is it alright to default to `z_index: 0` in this case?

Reply:
yes, it's fine